http/middleware.go

```diff
@@ -39,8 +39,20 @@ func WrapHandler(waf coraza.WAF, l Logger, h http.Handler) http.Handler {
 			return
 		}
 
+		i := &rwInterceptor{w: w, tx: tx, StatusCode: 200}
```

Comment:
Do we need to set the 200 status code?

Reply:
The idea was to set it to 200 as the default value at creation time. Internally the interceptor should be able to change it if an intervention will raise